frontend/src/actions/LoginThunkActions.ts

```diff
@@ -36,7 +36,10 @@ const performLogin = (dispatch: KialiDispatch, state: KialiAppState, data?: any)
         error => bail(error)
       );
     } else {
-      bail({ status: AuthResult.FAILURE, error: 'Preparation for login failed, try again.' });
+      bail({
+        status: AuthResult.FAILURE,
+        error: $t('tip153', 'Preparation for login failed, try again.')
```

Comment:
Can we set another name instead of tip? Something with more sense like 'login_failed_msg'

Reply:
I will sort out the translation later and try to make sure that I can understand the meaning of the corresponding expression when I see the name.